plugins/generator-1.15.2/forge-1.15.2/procedures/entity_set_sprinting.java.ftl

```diff
@@ -0,0 +1,3 @@
+if(${input$entity} instanceof LivingEntity){
```

Comment:
Is this check really needed since there is no casting performed afterwards?

Reply:
Yes the check is needed considering no one except the player can sprint. Btw the problem here is that I'm checking for LivingEntity and not PlayerEntity.

Reply:
It has been fixed! Btw if you check the code for player's gamemode, you'll notice there's also a check for PlayerEntity

Reply:
This is not how this should work. If any entity has this method it means that it "supports" it no matter if it can be visually seen in-game. By limiting this to only the player you are limiting the potential uses with other mods that perhaps implement sprinting on mobs other than the player. Checking whether it should make the target entity sprint is up to the procedure creator to check (not the procedure block).

Reply:
Then what do I do? I remove the check?

Reply:
Yes, the check here is no needed as noted by @MysteriousWolf

Reply:
Done! I removed the checks and fixed the merging issues.